codespell_lib/_codespell.py

```diff
@@ -43,6 +43,7 @@
     ('clear', 'for unambiguous errors', '', False, None),
     ('rare', 'for rare but valid words', '_rare', None, None),
     ('informal', 'for informal words', '_informal', True, True),
+    ('usage', 'for recommended terms', '_usage', True, None),
```

Comment:
```suggestion
    ('usage', 'for recommended terms', '_usage', None, None),
```
Should fix the CI error

Reply:
Oops sorry, my duff suggestion.

Reply:
The check passed locally with your suggestion @peternewman, which is why I applied it. Not sure what the difference is between the setups.

Reply:
> Not sure what the difference is between the setups.

Can we see your test output. it's probably down to the Aspell stuff. Try running it with `export REQUIRE_ASPELL=true` first.

Reply:
If I go with `    ('usage', 'for recommended terms', '_usage', True, None),` for the dictionary definition, and remove `man-in-the-middle` from the dictionary (it's not in aspell), the test passes cleanly.

My guess is that my aspell dictionaries (`aspell-en-2019.10.06-2.fc32.x86_64` on Fedora 32) are newer and contain a word that's not in the aspell dictionary that's in the CI.

Reply:
> If I go with ` ('usage', 'for recommended terms', '_usage', True, None),` for the dictionary definition, and remove `man-in-the-middle` from the dictionary (it's not in aspell), the test passes cleanly.
> 
> My guess is that my aspell dictionaries (`aspell-en-2019.10.06-2.fc32.x86_64` on Fedora 32) are newer and contain a word that's not in the aspell dictionary that's in the CI.

Yeah it was complaining about these:
```
E           error 'whitelist' should be in aspell for dictionary /home/travis/build/codespell-project/codespell/codespell_lib/tests/../data/dictionary_usage.txt
E           error 'whitelists' should be in aspell for dictionary /home/travis/build/codespell-project/codespell/codespell_lib/tests/../data/dictionary_usage.txt
```

Which is in the aspell dictionary:
http://app.aspell.net/lookup?dict=en_US&words=whitelists

Our CI is using:
https://packages.ubuntu.com/bionic/aspell-en
Which is 2017.08.24-0-0.1

So yeah you're just a bit newer!